### PR TITLE
Presentation: Support Binary/BeGuid type properties

### PR DESCRIPTION
**Pull request by medikornov**
Closes: https://github.com/orgs/iTwin/projects/20/views/1?pane=issue&itemId=22725591
native PR: https://github.com/iTwin/imodel-native/pull/254
imodel-native: https://github.com/iTwin/imodel-native/tree/presentation/support-guid-type-properties

### CODE REVIEW

File: full-stack-tests/presentation/src/frontend/Content.test.ts [hunk 1]
```diff
@@ -399,6 +399,44 @@ describe("Content", () => {
...
+              isDisplayed: true,
+              doNotHideOtherPropertiesOnDisplayOverride: true
+            }]
+          }],
+        }],
+      };
+      const content = await Presentation.presentation.getContent({
+        imodel: imodelConnection,
+        rulesetOrId: ruleset,
+        keys: new KeySet([{ className: instanceKey!.className, id: instanceKey!.id }]),
```
Comment (grigasp) [inline full-stack-tests/presentation/src/frontend/Content.test.ts:428]:
```suggestion
        keys: new KeySet([instanceKey!]),
```

File: full-stack-tests/presentation/src/frontend/Content.test.ts [hunk 2]
```diff
@@ -399,6 +399,44 @@ describe("Content", () => {
...
+      const ruleset: Ruleset = {
+        id: Guid.createValue(),
+        rules: [{
+          ruleType: RuleTypes.Content,
+          specifications: [{
+            specType: ContentSpecificationTypes.SelectedNodeInstances,
+            propertyOverrides: [{
+              name: "FederationGuid",
+              isDisplayed: true,
+              doNotHideOtherPropertiesOnDisplayOverride: true
```
Comment (grigasp) [inline full-stack-tests/presentation/src/frontend/Content.test.ts:420]: I believe in this test we only care about the `FederationGuid` property, so don't need others to be loaded too. In that case, the `doNotHideOtherPropertiesOnDisplayOverride` attribute is unnecessary.

**Review by grigasp — CHANGES_REQUESTED** [review]

File: full-stack-tests/presentation/src/frontend/Content.test.ts (hunk 1 above)
Comment (medikornov) [reply full-stack-tests/presentation/src/frontend/Content.test.ts]: fixed

File: full-stack-tests/presentation/src/frontend/Content.test.ts (hunk 2 above)
Comment (medikornov) [reply full-stack-tests/presentation/src/frontend/Content.test.ts]: fixed

**Review by grigasp — APPROVED** [review]

**Pull request merged**